### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by eduardo-8** [rustdesk/rustdesk#1147] [General]
![image](https://user-images.githubusercontent.com/30558853/182258326-5eae5ab9-5b73-40f7-a484-04a92156df50.png)

**Comment by rustdesk** [comment]
looks like you are controling yourself

**Reply by eduardo-8** [reply]
I've been here for 11 days

**Reply by chiehw** [reply]
Are your remote and local machines using the same screen?
